Cargo.toml

```diff
@@ -441,6 +441,7 @@ rustc-demangle = "0.1.23"
 rust-embed = { version = "8.4", features = ["include-exclude"] }
 rustls = "0.21.12"
 rustls-native-certs = "0.8.0"
+scap = "0.0.7"
```

Comment:
I just had a quick look at this crate and it is an unfortunate choice to use it, if it is for linux only. As you end up pulling a separate dbus library (instead of zbus that is already pulled by ashpd and oo7). I would probably suggest using ashpd directly for screencasting, see https://github.com/bilelmoussaoui/ashpd/blob/master/examples/screen_cast_pw.rs